CHANGELOG.md

```diff
@@ -1,4 +1,19 @@
-## 2.0.0
+# upcoming
+
+* Allow to use mutliple `.po` files
+* BREAKING CHANGE: files are now longer defined in `options`, use standard grunt `files` object:
+```js
+'gettext-checker': {
+    all: {
+        files: {
+            'i18n/template.pot': 'i18n/*.po'
```

Comment:
Can you explain here or in the readme that the DEST should be the catalog and the SRC should be the individual langauges?